.github/workflows/build-unsigned-ios.yml

```
@@ -0,0 +1,40 @@
+            CODE_SIGNING_ALLOWED=NO
+
+      # Package the archive into an IPA
+      - name: Create unsigned IPA
+        run: |
+          cd unsigned.xcarchive/Products
+          mv Applications Payload
+          zip -r unsigned.ipa Payload
+          mv unsigned.ipa "$GITHUB_WORKSPACE"
+
+      # Upload the IPA
+      - uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)